pgcli/main.py

```diff
@@ -520,8 +523,18 @@ def cli(database, user, host, port, prompt_passwd, never_prompt, dbname,
             '\thost: %r'
             '\tport: %r', database, user, host, port)
 
+    obfuscate_process_password()
     pgcli.run_cli()
 
+def obfuscate_process_password():
+    process_title = setproctitle.getproctitle()
+    if '://' in process_title:
+        process_title = re.sub(r":(.*):(.*)@", r":\1:xxxx@", process_title)
+    elif "=" in process_title:
+        process_title =  re.sub(r"password=(.+?)((\s[a-zA-Z]+=)|$)", r"password=xxxx\2", process_title)
```

Comment:
Nice! I was trying to think of a way to enhance the regex to address the concern I raised. But I couldn't come up with something reasonable. So I'm glad you were able to find an alternate regex that works.